roslibrust_test/src/lib.rs

```diff
@@ -0,0 +1,1885 @@
+#[allow(unused_imports)]
```

Comment:
fmt... I think I'll try to tackle that as a way to test these changes out.

Reply:
Okay so I did some digging and I hate things...

I personally have some major style gripes with this:

1. There aren't empty lines between blocks
2. the ":: serde :: Deserialize" feels like it should really be "::sered::Deserialize"

However neither of these appears fixable with a .rustfmt file so I'm going to have to live with it. I am at least adding a lint check to CI cause we don't have one.